### PR TITLE
Add an example to visualize and demonstrate 2d text alignment settings

### LINKED ISSUE #8994: Text2d API example
## What problem does this solve or what need does it fill?

The current Text2d example is fine as an example of its capabilities. But the way Text2d handles alignment and anchoring is very different to how text in Bevy UI works, and some users find the API difficult to understand.

## What solution would you like?

Just a simple app that spawns a `Text2dBundle` and some buttons that you can press to update the Text2d entity with different anchor and alignment settings would be fine.

Maybe buttons for `BreakLineOn` too, if it doesn't make things too busy.

### PR DESCRIPTION
**Pull request by Adamkob12**
# Objective

- Create an example to give a better demonstration of how to use the 2d text API.
- fixes #8994 

## Solution

- Add an example to demonstrate how to mutate the Anchor and alignment of text, and visualize how it works.

Per suggestion in discord

### CODE REVIEW

File: examples/2d/dynamic_text2d.rs [hunk 1]
```diff
@@ -0,0 +1,200 @@
+use bevy::prelude::*;
```
Comment (alice-i-cecile) [inline examples/2d/dynamic_text2d.rs:1]: Module level docs go here explaining what the example does.

File: examples/2d/dynamic_text2d.rs [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    }
+}
+
+fn move_anchor_point(
+    mut anchor_query: Query<&mut Transform, With<DynamicText>>,
+    keyboard_input: Res<Input<KeyCode>>,
+    time: Res<Time>,
+) {
+    let mut move_dir = Vec3::ZERO;
+    if keyboard_input.pressed(KeyCode::W) {
```
Comment (alice-i-cecile) [inline examples/2d/dynamic_text2d.rs:58]: Can you use arrow keys instead? They'll play nicer on non-English keyboard layouts.

**Comment by ickshonpe** [comment]
Could you add "fixes #8994" to the objective.

I'm sorry I didn't get around to looking at this earlier after I suggested it to you as a project. I'll do a full review later today.

**Comment by Adamkob12** [comment]
Sorry, kinda neglected the PR because I got busy with exams. Will pick it up in a couple of days.

**Pull request closed without merging**